app/src/main/java/com/fstyle/androidtrainning/screen/main/fragments/mysong/subfragment/album/AlbumFragment.java

```diff
@@ -0,0 +1,35 @@
+package com.fstyle.androidtrainning.screen.main.fragments.mysong.subfragment.album;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.screen.main.MainPresenter;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.screen.main.MainPresenter. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports